### ISSUE TITLE
Update for new Anycubic cloud endpoints?

### ISSUE THREAD

**Issue by polyphia** [Royrdan/anycubic_cloud#4]
Hi, any chance you ll be updating endpoints to work with anycubic new cloud ?

Would be extremely useful to enable actions triggered by octoprint such as spaghetti detection which works quite reliably .

**Comment by Royrdan** [comment]
Hello. I havnt been much printing lately so hadn't realised they have updated the app and API. I will have to reverse engineer the new app and will see if it's an easy fix. Thanks for letting me know.

**Comment by adamoutler** [comment]
Nope.  I have examined the device and found a responsible daemon, but no progress.

**Comment by adamoutler** [comment]
`ssh -oKexAlgorithms=+diffie-hellman-group1-sha1 root@192.168.1.137  'cd /usr/sbin/LOG/;tail -n1000 -f "$(ls -tr | head -n 1)" |grep msgcontent|sed "s/.*msgconten
t://g"'` will login and grab json to the terminal.  Looking for a way to make it work better.  Anycubic has not responded to a request I sent.

**Comment by danielecollins** [comment]
Any updates on this issue?
